.github/workflows/aks.yaml

```
@@ -59,7 +60,7 @@ jobs:
       rancher_version: ${{ inputs.rancher_version || '2.9-head' }}
       k3s_version: ${{ inputs.k3s_version || 'v1.28.9+k3s1' }}
       operator_nightly_chart: ${{ inputs.operator_nightly_chart == true || (github.event_name == 'schedule' && true) }}
-      tests_to_run: ${{ inputs.tests_to_run || 'p0_provisioning/p0_import' }}
+      tests_to_run: ${{ inputs.tests_to_run || (github.event.schedule == '0 4 * * 1-6' && 'p0_provisioning/p0_import') || (github.event.schedule == '0 4 * * 0' && 'p0_provisioning/p0_import/p1_provisioning/p1_import/sync_provisioning/sync_import') }}
```

```suggestion
      tests_to_run: ${{ inputs.tests_to_run || (github.event.schedule == '0 4 * * 1-6' && 'p0_provisioning/p0_import') || (github.event.schedule == '0 4 * * 0' && 'p1_provisioning/p1_import/sync_provisioning/sync_import') }}
```

Since we run p0 nightly, perhaps there's no need to run them on again?